README.md

```diff
@@ -83,8 +83,11 @@ You can also come chat with the community in the Matrix [room](https://matrix.to
 
 ## Build instructions
 
-Just clone the project and open it in Android Studio.
-Makes sure to select the `app` configuration when building (as we also have sample apps in the project).
+Just clone the project and open it in Android Studio. Make sure to select the
+`app` configuration when building (as we also have sample apps in the project).
```

Comment:
The sample project was deleted, I believe.

Reply:
(Yes, I have seen this, but it's out of scope of this PR, I'll handle this later)